modules/installation-vsphere-installer-infra-static-ip-nodes.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: CONCEPT
+[discrete]
+[id="installation-vsphere-installer-infra-static-ip-nodes_{context}"]
+== Static IP addresses for vSphere nodes
+
+You can provision bootstrap, control plane, and compute nodes to use static IP addresses, so that you can better configure your network to meet the security demands of your organization. To configure this environment, you must provide values to the `platform.vsphere.hosts.role` parameter in the `install-config.yaml` file.
+
+:FeatureName: Static IP addresses for vSphere nodes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+By default, the installation program is configured to use the Dynamic Host Configuration Protocol (DHCP) for the network, but this network has limited configurable capabilities.
+
+After you define one or more machine pools in your `install-config.yaml` file, you can define network definitions for nodes on your network. Ensure that the number of network definitions matches the number of machine pools that you configured for your cluster.
+
+The following example shows a `compute` network definition specified for the `role` parameter:
+
+[source,yaml]
+----
+---
+platform:
+  vsphere:
+    hosts:
+    - role: compute <1>
+    networkDevice:
+      ipAddrs:
+      - 192.168.204.10/24 <2>
+      gateway: 192.168.204.1 <3>
+      nameservers:
+      - 192.168.204.1 <4>
+---
+----
+<1> Valid network defintion values include `bootstrap`, `control-plane`, and `compute`. You must list at least one `bootstrap` network definition in your `install-config.yaml` configuration file.
+<2> Lists IPv4, IPv6, or both IP addresses that the installation program passes to the network interface. Both IP families must use the same network interface for the default network. 
+<3> The default gateway for the network interface. 
+<4> Lists up to 3 DNS servers. A DNS server can resolve any provided IP addresses by a process known as DNS resolution.
++
+[IMPORTANT]
+====
+To enable the Technology Preview feature of static IP addresses for vSphere nodes for your cluster, you must include `featureSet:TechPreviewNoUpgrade` as the initial entry in the `install-config.yaml` file.
+====
+
+After you deployed your cluster to run nodes with static IP addresses, you can scale a machine to use one of these static IP addresses. Additionally, you can use a machine set to configure a machine to use one of the configured static IP addresses.
```

Comment:
```suggestion
After you deploy your cluster to run nodes with static IP addresses, you can scale a machine to use one of these static IP addresses. Additionally, you can use a machine set to configure a machine to use one of the configured static IP addresses.
```

Reply:
I think capturing the past tense is essential here. "Deploy" would indicate the deployment operation is not finished, as is ongoing. Is this correct?

Reply:
correct